src/main/java/com/school/project/controller/GroupController.java

```diff
@@ -43,6 +50,7 @@ private Group convertGroupDtoToGroup(GroupDto groupDto) {
         return modelMapper.map(groupDto, Group.class);
     }
 
+
```

Comment:
why are you adding all this new lines across the project? pls double check all your changes and delete all double lines everywhere except imports (this zone is handled by IDE)